spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -176,10 +176,43 @@ def call_with_params(params)
     end
   end
 
+  describe "edit" do
+    let(:user) { create(:user) }
+    let(:work) { create(:work, authors: [user.default_pseud], posted: true) }
```

Comment:
The work factory creates posted works by default as of #3602, so `posted: true` shouldn't be necessary in most of these tests.

Reply:
Fixed